app/scripts/modules/core/src/pipeline/config/stages/deploy/deployStage.js

```diff
@@ -149,6 +149,7 @@ angular
                   templateUrl: config.cloneServerGroupTemplateUrl,
                   controller: `${config.cloneServerGroupController} as ctrl`,
                   size: 'lg',
+                  windowClass: 'modal-z-index-fix',
```

Comment:
Does this overwrite the other classes or is it supplementary? 

If this does not overwrite, then I would suggest using our existing styleguide classname `layer-critical` instead of this name. Then you will not need the update to `main.less`

Reply:
I think it adds it to the modal window template. When I use the `layer-critical` class the issue continues to exist.

Reply:
Did you mean `layout-critical`?

Reply:
No `layer-critical`

Reply:
if that doesn't work, can you just update the class to `modal-z-index` and remove the `fix`?

Reply:
Updated